app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -0,0 +1,56 @@
+<%
+  attachment = GovukPublishingComponents::Presenters::Attachment.new(attachment)
+  target ||= nil
+  hide_help_text ||= false
+  attributes = []
+
+  if attachment.content_type_abbr
+    content = tag.abbr(attachment.content_type_abbr,
+                       class: "gem-c-attachment__abbr",
+                       title: attachment.readable_content_type)
+    attributes << tag.span(content)
+  end
+
+  if attachment.readable_file_size
+    attributes << tag.span(attachment.readable_file_size)
+  end
+
+  if attachment.readable_number_of_pages
+    attributes << tag.span(attachment.readable_number_of_pages)
+  end
+%>
+<%= tag.section class: "gem-c-attachment" do %>
+  <%= tag.div class: "gem-c-attachment__thumbnail" do %>
+    <%= link_to attachment.url, target: target do %>
+      <%= tag.img class: "gem-c-attachment__thumbnail-image",
+                  src: attachment.thumbnail,
+                  alt: attachment.title
+      %>
+    <% end %>
+  <% end %>
+
+  <%= tag.div do %>
+    <%= tag.h2 class: "gem-c-attachment__title" do %>
+      <%= link_to attachment.title, attachment.url,
+            class: "govuk-link",
+            target: target %>
+    <% end %>
+    <%= tag.p class: "gem-c-attachment__metadata" do %>
+      <%= raw(attributes.join(', ')) if attributes.any? %>
+    <% end %>
+  <% end %>
+
+  <% unless hide_help_text %>
+    <%= tag.div do %>
```

Comment:
Is this `div` necessary? It's not styled and doesn't seem to serve any purpose. Also it's outside of the logic below so it can appear empty (see first example in component guide).

Reply:
Now removed